mobile-development/mobile-development.md

```diff
@@ -14,6 +14,18 @@ About 4 hours
 
 ### Motivation
 
+Mobile applications have revolutionized Mobile World, getting much more power in the hands of customer & more business in the pockets of the entrepreneurs. In the dynamic technology industry, mobile application development market is at its peak. Its is the need of the hour & the latest fruitful business trend.[(by Sara bhavsar)](https://www.quora.com/Why-is-the-mobile-application-development-important)
```

Comment:
Can you either:
1. put this section in quotes to show these are Sara's exact words
or
2. Fix the grammar - its a bit weird.

Reply:
Oh and capitalize her last name, please. Also on line 24.